docs/en/ingest-management/faq.asciidoc

```diff
@@ -227,3 +226,29 @@ and port for your setup. If you run everything locally, the address is
 you can copy the {es} endpoint URL from the overview page of your deployment.
 If you're not running in {ecloud}, make sure the {kib} and {es} HTTPS ports
 are both accessible; by default these are `5601` and `9200` respectively.
+
+[discrete]
+[[how-do-i-reset-a-missing-dashboard-asset]]
+== If I delete an integration dashboard asset from {kib}, how do I get it back?
+
+To reinstall the assets for a specific integration, you can use the UI for most cases
+and the {fleet} API for those not allowed in the UI.
+In order to use the API, you'll need to know the package name and version
+you want to reinstall.
+To reinstall package assets, execute the following call with the `force` parameter in the body:
+
+[source,console]
+----
+POST /api/fleet/epm/packages/[package name]-[package version]
+{ "force": true }
+----
+
+So, for example, to reinstall the system v1.0.0 package, POST:
+[source,console]
+----
+POST /api/fleet/epm/packages/system-1.0.0
+{ "force": true }
+----
+
+The package version is shown in the Integrations view in {kib} or can be ascertained 
+using the https://epr.elastic.co/search?package=system[registry] 
```

Comment:
```suggestion
The package version is shown in the Integrations view in {kib}.
```

do we reference epr.elastic.co anywhere else in docs? if not, I think we should hide this complexity from the user

Reply:
I was thinking we need to start exposing it somewhere, if it isn't already.  I was trying to sneak it in, but it isn't really needed for this topic.  I could start a new one, but first I can review if we have it exposed or not, and we can figure out if we want to start with this (we may not, as you note).